fbs.features.field_instance.inc

```diff
@@ -246,10 +241,9 @@ function fbs_field_default_field_instances() {
     'display' => array(
       'default' => array(
         'label' => 'above',
-        'module' => 'fbs',
         'settings' => array(),
-        'type' => 'fbs_preferred_branch',
-        'weight' => 7,
+        'type' => 'hidden',
+        'weight' => 2,
       ),
     ),
     'entity_type' => 'profile2',
```

Comment:
There are a number of fields that are related to he user profile that got "hidden", like prefferred branch, interest period. etc. I assume these fields won't be displayed for the end user that tries to edit his profile. This might not be what is expected and must be checked with a test user.